Gaming The Syllabus/Assets/Scripts/Combat/Action Effects/DamageTarget.cs

```diff
@@ -25,6 +25,10 @@ public override void Apply(params CombatUnit[] targets)
         foreach (CombatUnit target in targets)
         {
             target.ApplyDamage(damageAmount);
+            CombatUnit attacker = CombatManager.getCurrentCombatant();
+            DialogueBoxUIManager.addStringToDialogueBox
```

Comment:
This seems like a pain to write every time we wanna do this. Maybe we can come up with a method on DialogueBoxUIManager to support coloring the attacker and defender? Not entirely sure how that would be structured atm but something to consider